### PR TITLE
add lottie

### PR DESCRIPTION
**Pull request by stavros-k**
**Description**

⚒️ Fixes  # 

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**

**📃 Notes:**

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning
- [ ] I made sure the title starts with `feat(chart-name):`, `fix(chart-name):` or `chore(chart-name):`

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

### CODE REVIEW

File: astro.config.mjs [hunk 1]
```diff
@@ -105,8 +108,8 @@ export default defineConfig({
       sidebar: [
         {
           label: "General",
-          collapsed: false,
-          autogenerate: { directory: "general" },
+          collapsed: true,
+          autogenerate: { directory: "general", collapsed: true },
```
Comment (Ornias1993) [inline astro.config.mjs:112]: doesn't one need ot be true and one false here?

File: src/components/CustomHero.astro [hunk 2]
```diff
@@ -0,0 +1,164 @@
+---
+import { Image } from "astro:assets";
+import { PAGE_TITLE_ID } from "../../node_modules/@astrojs/starlight/constants";
+import type { Props } from "../../node_modules/@astrojs/starlight/props";
+import CallToAction from "../../node_modules/@astrojs/starlight/components/CallToAction.astro";
```
Comment (Ornias1993) [inline src/components/CustomHero.astro:5]:
```suggestion
import { PAGE_TITLE_ID } from "@astrojs/starlight/constants";
import type { Props } from "@astrojs/starlight/props";
import CallToAction from "@astrojs/starlight/components/CallToAction.astro";
```

Comment (stavros-k) [reply src/components/CustomHero.astro:5]: It didnt seem to "resolve" on editor, let me actually test if it works

File: astro.config.mjs (hunk 1 above)
Comment (stavros-k) [reply astro.config.mjs:112]: interstingly, it works with this, but also with false and true. which makes more sense

File: src/components/CustomHero.astro (hunk 2 above)
Comment (Ornias1993) [reply src/components/CustomHero.astro:5]: thats how you reference on all the other custom components

File: astro.config.mjs (hunk 1 above)
Comment (Ornias1993) [reply astro.config.mjs:112]: yeah likely a bug...
So lets go for the false/true

**Pull request merged**